config/locales/zh.yml

```diff
@@ -1096,7 +1095,7 @@ idv.messages.gpo.start_over_html: 如果该地址不对，你需要%{start_over_
 idv.messages.gpo.start_over_link_text: 重新开始并用你新地址进行验证。
 idv.messages.gpo.timeframe_html: 你会在<strong> 5 到 10 天</strong> 里收到带有<strong>验证码</strong> 的信。
 idv.messages.otp_delivery_method_description: 如果你在上面输入的是座机电话，请在下边选择“接听电话”。
-idv.messages.phone.alert_html: '<strong>输入一个这样的电话号码：</strong> '
+idv.messages.phone.alert_html: '<strong>输入一个这样的电话号码：</strong>'
```

Comment:
This was an inconsistency compared to the other languages and seems to have been a mistake.